src/Lean/Server/Rpc/RequestHandling.lean

```diff
@@ -87,21 +95,17 @@ def registerBuiltinRpcProcedure (method : Name) paramType respType
   let proc := wrapRpcProcedure method paramType respType handler
   builtinRpcProcedures.modify fun ps => ps.insert method proc
 
-def registerRpcProcedure (method : Name) paramType respType
-    {paramLspType} [RpcEncoding paramType paramLspType] [FromJson paramLspType]
-    {respLspType} [RpcEncoding respType respLspType] [ToJson respLspType]
-    (handler : paramType → RequestM (RequestTask respType)) : CoreM Unit := do
+open Lean.Elab.Command in
+def registerRpcProcedure (method : Name) : CommandElabM Unit := do
   let env ← getEnv
-
   let errMsg := "Failed to register RPC call handler for '{method}'"
   if (←builtinRpcProcedures.get).contains method then
     throwError s!"{errMsg}: already registered (builtin)"
-  if userRpcProcedures.getState env |>.contains method then
+  if userRpcProcedures.contains env method then
     throwError s!"{errMsg}: already registered"
```

Comment:
This might cause trouble. When two imported modules register the same name, do both `registerRpcProcedure` calls succeed and store the same name in extension state? At what point do we see a conflict?